cypress/e2e/feeder-rss-import.cy.ts

```diff
@@ -37,6 +37,29 @@ describe("Feeder RSS Import", () => {
     cy.contains(".badge", "Complete", { timeout: 60000 });
     cy.contains("00:00:10");
 
+    // test timings import
+    const timingsFile = "cypress/samples/timings.csv";
```

Comment:
Given the nature of this data, I wonder if it would be worthwhile in this test to also test some invalid data to ensure that is handled as expected also.

Reply:
There is some error handling built into the timings importer ... I think if you just add a line to the CSV with a guid that doesn't exist in the database, you'll get an EpisodeImport in status = error.